src/formatter.ts

```diff
@@ -123,6 +123,16 @@ export class JupyterlabNotebookCodeFormatter extends JupyterlabCodeFormatter {
       }
     }
 
+    // finally, try to get the language from the current session's kernel spec
```

Comment:
Is it intended that this block will not be reached when metadata are not defined?